Rsite/source/tutorial/CustomLossGluon.md

```diff
@@ -0,0 +1,235 @@
+
+# Custom Loss functions in Gluon
+
+Loss functions ared used to train a neural network and to compute the difference between output and target variable. Consequently, they are a key element to successfully train models. Depending on the model, we may choose one or another function. MXNet's Gluon API provides most commonly used functions. For instance:
+
+- regression: [L1Loss](https://beta.mxnet.io/api/gluon/_autogen/mxnet.gluon.loss.L1Loss.html), [L2Loss](https://beta.mxnet.io/api/gluon/_autogen/mxnet.gluon.loss.L2Loss.html) 
+- classification: [SigmoidBinaryCrossEntropyLoss](https://beta.mxnet.io/api/gluon/_autogen/mxnet.gluon.loss.SigmoidBinaryCrossEntropyLoss.html), [SoftmaxBinaryCrossEntropyLoss](https://beta.mxnet.io/api/gluon/_autogen/mxnet.gluon.loss.SoftmaxBinaryCrossEntropyLoss.html) 
+- embeddings: [HingeLoss](https://beta.mxnet.io/api/gluon/_autogen/mxnet.gluon.loss.HingeLoss.html)
+
+However, we may sometimes want to solve problems that require customized loss functions and this tutorial shows how we can do that in Gluon. We will show it at the example of contrastive loss which is typically used in Siamese networks.
+
+```python
+import matplotlib.pyplot as plt
+import mxnet as mx
+from mxnet import autograd, gluon, nd
+from mxnet.gluon.loss import Loss
+import random
+```
+
+### What is contrastive Loss
+
+[Contrastive loss](http://yann.lecun.com/exdb/publis/pdf/hadsell-chopra-lecun-06.pdf) is a distance-based loss function. During training pairs of images are fed into a model. If the images are similar the loss function will return 0, otherwise 1. 
```

Comment:
If the images are similar, the loss function will return 0, otherwise it will return 1.
(adding commas and expanding the contraction)